src/widgets/BreachChart.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
as the top level comment says, created this because it seemed just nicer than having a bunch of conditionals in the `VaccineChart`, eespecially with the change to colours coming from a `TemperatureBreachConfiguration`